OpenAnatomyExport/OpenAnatomyExport.py

```diff
@@ -499,6 +511,28 @@ def addModelToRenderer(self, inputModelNode, outputModelNode, boostGouraudColor=
 
     return True
 
+  def createPlaneModelFromMarkupsPlane(self,planeMarkup):
+    planeBounds = planeMarkup.GetPlaneBounds()
+    objectToWorld = vtk.vtkMatrix4x4()
+    planeMarkup.GetObjectToWorldMatrix(objectToWorld)
+
+    # Create plane polydata
+    planeSource = vtk.vtkPlaneSource()
+    planeSource.SetOrigin(objectToWorld.MultiplyPoint([planeBounds[0], planeBounds[2], 0.0, 1.0])[0:3])
+    planeSource.SetPoint1(objectToWorld.MultiplyPoint([planeBounds[1], planeBounds[2], 0.0, 1.0])[0:3])
+    planeSource.SetPoint2(objectToWorld.MultiplyPoint([planeBounds[0], planeBounds[3], 0.0, 1.0])[0:3])
+    planeModel = slicer.modules.models.logic().AddModel(planeSource.GetOutputPort())
+
+    # Copy props from markups to model
+    planeMarkupDisplayNode = planeMarkup.GetDisplayNode()
+    planeModelDisplayNode = planeModel.GetDisplayNode()
+    planeColor = planeMarkupDisplayNode.GetSelectedColor()
```

Comment:
No need to create a variable if it is only used once and the result of a simple Get function.

Instead of this:

```
planeColor = planeMarkupDisplayNode.GetSelectedColor()
planeModelDisplayNode.SetColor(planeColor)
```

This is simpler and safer (less chance for copy-paste errors):

```
planeModelDisplayNode.SetColor(planeMarkupDisplayNode.GetSelectedColor())
```